### PR TITLE
Restore `hide-zero-packages`

### LINKED ISSUE #3572: re-activate `hide-zero-packages`
I noticed that repository sidebars lists empty Packages sections ( see screenshot attached for the Refined GitHub repository ). I was unaware that we previously had a feature that hid zero packages section and it was dropped in #3297. I don't know if this is only me but I see the Packages section back again. Maybe something changed at GitHub? Should we bring the feature back again?

image

### PR DESCRIPTION
**Pull request by kidonng**
Thanks for contributing! 🍄

1. LINKED ISSUES:
   Close #3572

   > [Just to mention that if we want to hide “Packages” tab on organization, we need to query the API.](https://github.com/sindresorhus/refined-github/pull/3297#issuecomment-650671415)

2. TEST URLS:
   https://github.com/sindresorhus/refined-github (no packages)
   https://github.com/yarnpkg/berry (with packages)
   https://github.com/sindresorhus (no packages)
   https://github.com/kidonng (with packages)

3. SCREENSHOT:

### CODE REVIEW

File: source/features/hide-zero-packages.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+void features.add({
+	id: __filebasename,
+	description: 'Hides the `Packages` tab if it’s empty (in repositories and user profiles).',
+	screenshot: 'https://user-images.githubusercontent.com/44045911/93541991-80354800-f98a-11ea-8347-a571abbbd318.png'
+}, {
+	include: [
+		pageDetect.isRepoRoot,
+		pageDetect.isUserProfile
+	],
+	waitForDomReady: false,
```
Comment (yakov116) [inline source/features/hide-zero-packages.tsx:29]:
```suggestion
	awaitDomReady: false,
```

**Comment by fregante** [comment]
The screenshot is a little hard to follow. It’s best if you can have the BEFORE/AFTER text above each column rather than between them

**Review by fregante — APPROVED** [review]

**Comment by kidonng** [comment]
> The screenshot is a little hard to follow. It’s best if you can have the BEFORE/AFTER text above each column rather than between them

Makes sense

**Comment by fregante** [comment]
9CB4492F-F0A1-4178-9318-01080E20B5BE
